PalindromeLab/README.md

```diff
@@ -0,0 +1,24 @@
+<h1>Python Autograder Template with Palindrome Lab</h1>
+
+<h2>Assessment Language</h2>
+Python
+
+<h2>Autograder Language</h2>
+Python
+
+<h2>Autograding Environment Packages</h2>
+Python3 and pip are needed for the autograding environment.
+
+<h2>Assessment Scenario</h2>
+In this lab, students are required to implement a function palindrome() 
+which takes in an integer as input.
+The function checks whether the integer is a palindrome
+(the number is the same whether it is read backwards or forwards).
+
+<h2>Hand-in Format</h2>
+palindrome.py
+
+<h2>Autograder.tar directory content</h2>
```

Comment:
You need a newline after the h2 tags for the list to render properly.

In the future, you can verify the syntax by viewing the README on your branch (https://github.com/autolab/autograders-examples/blob/kester/PalindromeLab/README.md), then start editing it and use the preview feature

Reply:
okay added a newline